server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -84,6 +84,9 @@
                         "[true, false, checksum] but was: " + s);
             }
         }, Property.IndexScope);
+    // This setting is undocumented as it is considered as an escape hatch.
+    public static final Setting<Boolean> ON_HEAP_ID_TERMS_INDEX =
+            Setting.boolSetting("index.force_memory_id_terms_dictinary", false, Property.IndexScope);
```

Comment:
nit: dictinary -> dictionary

Reply:
Thanks for catching this, you were not nit-picking!